src/cli/commands/__tests__/daemon.test.ts

```
@@ -0,0 +1,145 @@
+    sendDaemonStringRequestMock.mockRejectedValue(new Error('connect ENOENT'));
+
+    await handleDaemon('stop', [], false);
+
+    expect(logLines.join('\n')).toContain('wmux daemon is not running.');
+  });
+});
+
+describe('wmux daemon start', () => {
+  it('reports the spawned pid and pipe on a fresh start', async () => {
+    ensureDaemonMock.mockResolvedValue({
+      pid: 555, authToken: 'super-secret-token', pipeName: '\\.\pipe\wmux-daemon-x', spawned: true,
```

ESLint

[error] Unnecessary escape character: \p.

(no-useless-escape)